source/agora/common/SCPHash.d

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+
+    Contains definition of hashing routines which SCP uses.
+    These just call the hashing routines from agora.common.Hash
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.common.SCPHash;
+
+import agora.common.Hash;
+
+import scpd.types.Stellar_SCP;
+import scpd.types.Stellar_types;
+import scpd.types.Utils;
+
+extern(C++, `stellar`):
+
+/*******************************************************************************
+
+    Implementation of the hashing routines as used by SCP
+
+    Params:
+        qset = the SCP quorum set to hash
+
+    Returns:
+        the 64-byte hash
+
+*******************************************************************************/
+
+public uint512 getHashOf (ref const(SCPQuorumSet) qset)
+{
+    return uint512(hashFull(qset));
+}
+
+/// ditto
```

Comment:
It would be nice if D is uppercase.

Reply:
Okay! It doesn't make a difference to the compiler, but sure.

Reply:
Ah for consistency, sure.